src/tedpy/dataclasses.py

```diff
@@ -67,6 +67,8 @@ class MtuNet(NamedTuple):
 
     type: MtuType
     now: int
+    tdy: int
```

Comment:
DOH! You are totally right on matching the names. Don't know how I missed that... At a minimum I will rename those.

As far as making them inherit that might be a bit tricky since an MTU data could be Consumption or Production. Maybe (i'm not good with naming classes but...) base class of EnergyYield which Consumption and Production extend then MtuType contains an EnergyYield object?

Reply:
Part 1 until a path forward is decided: 29a89a8

Reply:
`EnergyYield` sounds great! Maybe have just two classes: `EnergyYield` and `MtuType` extending `EnergyYield`, then have a type field of type enum like you already have for `MtuType`.

Reply:
Have a look at this: 8cfecb4

I prefer Composition to Inheritance, gang of four is drilled into my head. The other note is Spyders are consumption only units. The documentation states they should not be used for production, however it then immediately follows by saying you can set the multiplier to negative for production. So... I'm just going to go with Consumption in the print-out at this time. I think this will set you up for grabbing System-Load and System-Generation pretty easily as well.

Reply:
This is reasonable. I think in the future I'd like to rework the design of the module so that `TedMtu` and `TedSpyder` both have `energy()` methods, which can return the `EnergyYield` struct. But for now this is perfect!